### PR TITLE
[ERP-2055] Apply all components

### PR DESCRIPTION
**Pull request by NikuPAN**
Put everything together, made necessary changes to components

### CODE REVIEW

File: public/site.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
...SI1Ni42OSIgeT0iNTYuNjciIHdpZHRoPSIxMTMuMzkiIGhlaWdodD0iMTEzLjM5Ii8+PC9jbGlwUGF0aD48L2RlZnM+PHRpdGxlPlFVVF9TUVVBUkVfUkdCX1JFVjwvdGl0bGU+PHJlY3QgY2xhc3M9ImNscy0yIiB3aWR0aD0iMTEzLjM5IiBoZWlnaHQ9IjExMy4zOSIvPjxnIGNsYXNzPSJjbHMtMyI+PHBhdGggY2xhc3M9ImNscy00IiBkPSJNMTA2LjQxLDY4czAsMjkuMDUsMCwzMC4zN2MwLDIuMTQsMi42MiwxMiwxNi4xOCwxMnMxNS43My05LjgsMTUuNzMtMTEuODhWODFsLTEwLjQtMTAuODlWOTcuMjhjMCwuNTUsMCwzLjMxLTUuMzYsMy4zMS01LjYxLDAtNS42MS0yLjcyLTUuNjEtMy4zOFY2OFoiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01Ni42OSAtNTYuNjcpIi8+PC9nPjxwb2x5Z29uIGNsYXNzPSJjbHMtNCIgcG9pbnRzPSI3Mi4yOSAxMS4zIDgyLjIgMjEuNTYgODkuMzcgMjEuNTYgODkuMzcgNTIuOTggMTAwLjIgNTIuOTggMTAwLjIgMjEuNTYgMTA4LjU2IDIxLjU2IDEwOC41NiAxMS4yOSA3Mi4yOSAxMS4zIi8+PGcgY2xhc3M9ImNscy0zIj48cGF0aCBjbGFzcz0iY2xzLTQiIGQ9Ik0xMDAuOTMsMTAwLjdsOC44Niw5SDk3Ljg1TDc4LjM3LDg5LjQyaDExLjVsMy4wNywzLjE1YTkuNDQsOS40NCwwLDAsMCwuNzQtMy45NSwxMS4wNywxMS4wNywwLDEsMC0yMi4xMy4xNSwxMSwxMSwwLDAsMCwxMS4wNiwxMSw2LjI0LDYuMjQsMCwwLDAsMi0uMjJjLjEsMCw4LDguNTMsOCw4LjU3LTIuMjYsMS42MS02LjU5LDIuNDItMTAsMi40Mi0xMS44NiwwLTIxLjA5LTEwLjExLTIxLjA5LTIxLjYxYTIxLjQzLDIxLjQzLDAsMCwxLDQyLjg1LjQ0LDIyLjM1LDIyLjM1LDAsMCwxLTMuNDQsMTEuMzUiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01Ni42OSAtNTYuNjcpIi8+PC9nPjwvc3ZnPg=="
```
Comment (ryan-bennett) [inline public/site.json]: We will need to use a different version of the QUT logo that works with a white background. You should be able to find one at the QUT brand centre https://brand-centre.qut.edu.au/, Go to Asset Library -> Logos after you login. Once you find a suitable logo, convert it to base 64 and update the reference here.

File: src/components/branding/Branding.tsx [hunk 2]
```diff
@@ -13,14 +13,15 @@ export interface BrandingProps extends HeadingProps {
...
 
 export const Branding: FunctionComponent<BrandingProps> = (props) => {
   const {
     logo,
     heading,
     hasLogoHeadingDivider = true,
-    size = ["sm", "md"],
+    size = props ? props?.size : ["sm", "md"],
     ...headingProps
   } = props;
```
Comment (ryan-bennett) [inline src/components/branding/Branding.tsx]: You can revert the change above and initialise the props i.e.  } = props || {};

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,17 +1,72 @@
...
 import Head from "next/head";
 import site from "../../public/site.json";
 import Page from "../components/page/Page";
 
+import { PlannedMaintenance } from "../components/plannedMaintenance";
+import { OperationalStatus, Service } from "../components/OperationalStatus";
+import { PastIncidents } from "../components/pastIncidents";
+import { ServiceStatus } from "../components/OperationalStatus/OperationalStatus";
+
+import statusJsonData from "../../public/status.json";
```
Comment (ryan-bennett) [inline src/pages/index.tsx]: These are compile time imports which is fine for the incidents and planned maintenance. The status data should be fetched in a useEffect hook

File: src/pages/index.tsx [hunk 4]
```diff
@@ -1,17 +1,72 @@
...
+import incidentsJsonData from "../../public/incidents.json";
+import plannedMaintenanceJsonData from "../../public/planned_maintenance.json";
+
 const Home: NextPage = () => {
+  const statusData = statusJsonData;
```
Comment (ryan-bennett) [inline src/pages/index.tsx]: const [statusData, setStatusData] = useState<Insert Status Data Type Here>();

useEffect(() =>> {
// fetch the status data and then poll every 60 seconds for new data
}, [])

**Review by ryan-bennett — CHANGES_REQUESTED** [review]
Looking good a couple of changes needed

Would also be worthwhile that to add a github action that just runs the yarn build. You can run this on every commit to ensure breaking changes can be checked prior to merge. Raise a seperate ticket if you don't want to cover ti under this commit

File: public/site.json (hunk 1 above)
Comment (NikuPAN) [reply public/site.json]: Updated

File: src/pages/index.tsx (hunk 3 above)
Comment (NikuPAN) [reply src/pages/index.tsx]: Updated

File: src/pages/index.tsx (hunk 4 above)
Comment (NikuPAN) [reply src/pages/index.tsx]: Updated

File: src/components/branding/Branding.tsx (hunk 2 above)
Comment (NikuPAN) [reply src/components/branding/Branding.tsx]: Updated

**Review by ryan-bennett — APPROVED** [review]
